xls2xml/xls2xml/utils.py

```diff
@@ -19,3 +23,135 @@ def header_to_xml_tag(header):
     # cannot contain spaces (same as above)
 
     return tag
+
+def validate_file(reader, validation_schema_filename):
```

Comment:
Returning the errors and printing them would be useful.

Reply:
It returns a boolean to indicate wether all the data pass the validation. The MetadataValidator prints the detailed error message when it comes across an error.